oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/dsl/ExpressionParsingContext.java

```diff
@@ -72,6 +72,8 @@ static ExpressionParsingContext create() {
 
     ScopeType scopeType;
 
+    boolean isRetagByK8sMeta;
```

Comment:
Please add a comment about what does this mean.

Reply:
done